action_types.xml

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<action_types>
+    <action name="act_equip_crossbow_light" type="actt_equip_unequip"/>
+    <action name="act_equip_crossbow_left_stance_light" type="actt_equip_unequip"/>
+    <action name="act_equip_crossbow_heavy" type="actt_equip_unequip"/>
+    <action name="act_equip_crossbow_left_stance_heavy" type="actt_equip_unequip"/>
```

Comment:
Is `actt_equip_unequip` a typo?

Reply:
> Is `actt_equip_unequip` a typo?
I think it's their abbreviation for action_type, below are the native entries, i had to double check as well although I copy pasted :D

Native:
`<action name="act_equip_crossbow" type="actt_equip_unequip"/>
<action name="act_equip_crossbow_left_stance" type="actt_equip_unequip"/>`

should be all good.